circuits/cpp/src/aztec3/circuits/abis/signed_tx_request.hpp

```diff
@@ -56,6 +57,16 @@ template <typename NCT> struct SignedTxRequest {
 
         return signed_tx_request;
     };
+
+    fr hash() const
+    {
+        // TODO: This is probably not the right thing to do here!!
```

Comment:
My guess is we'll want to concatenate all fields from tx_request and the signature and then hash (so we don't need to do two hashes), similar to how Ethereum does.